templates/broker.unit.erb

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Apache Kafka server (broker)
+Documentation=http://kafka.apache.org/documentation.html
+Requires=network.target remote-fs.target
+After=network.target remote-fs.target
```

Comment:
Removed the remote-fs. Zookeeper is needed, but it normally does not run on the same machine.